enterprise/utils.py

```diff
@@ -707,3 +694,66 @@ def is_course_run_enrollable(course_run):
     return (not end or end > now) and \
            (not enrollment_start or enrollment_start < now) and \
            (not enrollment_end or enrollment_end > now)
+
+
+def is_course_run_upgradeable(course_run):
+    """
+    Return true if the course run has a verified seat with an unexpired upgrade deadline, false otherwise.
+    """
+    now = datetime.datetime.now(pytz.UTC)
+    for seat in course_run.get('seats', []):
+        if seat.get('type') == 'verified':
```

Comment:
should we also check for professional and no_id_professional?

Reply:
Hmm, no, not here, but maybe somewhere else.

Reply:
`professional` and `no_id_professional` courses will be handled appropriately [here](https://github.com/edx/edx-enterprise/pull/279/files#diff-ce3a6d48e7f42961c07cd219f505d179R737). 
Basically, no "upgradeable" course runs will be found because professional/no_id_professional course runs are never "upgradeable" and we will default to including all course runs in the calculation to determine the current course run.